modules/20-arithmetics/27-commutativity/description.en.yml

```diff
@@ -1,7 +1,22 @@
 ---
 
 name: Commutative operations
-theory: 
-instructions: 
-tips:
+theory: |
+  Do you remember the basic arithmetic rule «changing the order of the numbers we are adding doesn't change the sum»? It is called the **commutative law**.
+
+  A binary operation is commutative, since you swap operands and you get the same result. Obviously, addition is a commutative operation: _3 + 2 = 2 + 3_.
```

Comment:
A binary operation is commutative, since you swap operands and you get the same result => 
A binary operation is commutative since swapping operands get you the same result.